python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -123,7 +123,7 @@ def with_updates(
         )
 
         return AssetReconciliationCursor(
-            latest_storage_id=latest_storage_id,
+            latest_storage_id=latest_storage_id or self.latest_storage_id,
```

Comment:
I know the code currently returns `latest_storage_id=None` or `latest_storage_id >= self.latest_storage_id`, but thoughts on making this a `max` function to guard against future code changes/improve readability?